### ISSUE TITLE
Deye Atom 3P config 

### ISSUE THREAD

**Issue by clowrey** [fonske/Modbus-configurations-esphome#2]
I am very curious how this config is working for you? and what model of inverter you are using it with? I am just compiling it now to try out with a Deye 25K 3P HV

https://github.com/fonske/Modbus-configurations-esphome/blob/main/deye/atoms3_lite_rs485/sundye_3P_1_atom.yaml

**Comment by clowrey** [comment]
Ahh this is so awesome I love ESPhome.. Thanks! I adjusted the voltage scaling to make it display the correct value and looks like a few other things need to be adjusted but overall its working :) Next I will multiply the "battery output power" by 10 to make it display the correct value too.. 

![image](https://github.com/fonske/Modbus-configurations-esphome/assets/6935928/63f3d470-e5c7-4295-8f0c-a8793ba564e2)

**Comment by fonske** [comment]
Hi.

I dont own a deye myself but i now some people who have one.
Can you let me know the adjustments you did? 
So i can keep the code up to date.

Have fun with it.

**Comment by clowrey** [comment]
Yup I will put it up on GitHub and link you to it here once I've got it all working, may be a few months before the whole system is online.

**Comment by Januszzlp** [comment]
@clowrey Did you manage to adjust the code ? I have one problem with the register 502 ( reactive power generation of
today ). Are you reading it correctly ?

**Comment by clowrey** [comment]
> @clowrey Did you manage to adjust the code ? I have one problem with the register 502 ( reactive power generation of today ). Are you reading it correctly ?

I am reading it, but the code is not currently running I am still setting up the rest of the system, so I cant test those values now.. But let me know if you figure out how to fix it :)
